src/software/embedded/constants/py_constants.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum, Flag
+
+import ipaddress
+
+
+class RobotPlatform(Enum):
```

Comment:
any reason to pybind the constants from `platform.h`

Reply:
Can't use pybind in `tbots.py`